doc/source/examples/cd_online_camelyon.ipynb

```diff
@@ -11,13 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook demonstrates a typical workflow for applying online drift detectors to streams of image data. For those unfamiliar with how the online drift detectors operate in `alibi_detect` we recommend first checking out the more introductory example [Online Drift Detection on the Wine Quality Dataset](https://docs.seldon.io/projects/alibi-detect/en/latest/examples/cd_online_wine.html) where online drift detection is performed for the wine quality dataset.\n",
-    "\n",
-    "Install the `wilds` library to fetch the dataset used in the example:\n",
-    "\n",
-    "```bash\n",
-    "pip install wilds\n",
-    "```"
+    "This notebook demonstrates a typical workflow for applying online drift detectors to streams of image data. For those unfamiliar with how the online drift detectors operate in `alibi_detect` we recommend first checking out the more introductory example [Online Drift Detection on the Wine Quality Dataset](https://docs.seldon.io/projects/alibi-detect/en/latest/examples/cd_online_wine.html) where online drift detection is performed for the wine quality dataset."
```

Comment:
Removed the wilds package install instructions?

Reply:
Only because its already in the explicit pip install below, see here for rendered example:

https://github.com/SeldonIO/alibi-detect/blob/52f542301c9d945f768728fc627ba497e882b785/doc/source/examples/cd_online_camelyon.ipynb

Reply:
p.s. just noticed typo in `torchvision`, will fix!